package.json

```diff
@@ -6,6 +6,12 @@
     "type": "git",
     "url": "git://github.com/greypants/gulp-starter.git"
   },
+  "scripts": {
+    "watch": "./node_modules/.bin/gulp",
+    "production": "./node_modules/.bin/gulp production",
+    "iconFont": "./node_modules/.bin/gulp iconFont",
+    "karma": "./node_modules/.bin/karma"
+  },
```

Comment:
`./node_modules/.bin/` is not needed.

https://docs.npmjs.com/cli/run-script:

> n addition to the shell's pre-existing PATH, npm run adds node_modules/.bin to the PATH provided to scripts. Any binaries provided by locally-installed dependencies can be used without the node_modules/.bin prefix.

You can rewrite them into:

```
"scripts": {
  "watch": "gulp",
  "production": "gulp production",
  "iconFont": "gulp iconFont",
  "karma": "karma"
}
```